flatten-data-ghcn_d.ipynb

```diff
@@ -0,0 +1,1167 @@
+{
```

Comment:
<p>Style nitpick:</p><p>If your notebook is using the Python3 kernel, you can use<a href="https://www.python.org/dev/peps/pep-0498/#supporting-full-python-expressions" target="_blank"> f-strings </a>for string interpolation. Rather than setting the query and formatting it you can just do this</p><pre class="ql-syntax" spellcheck="false">query = f"""
SELECT DISTINCT
&nbsp;base.id,&nbsp;
&nbsp;base.date,
&nbsp;stations.name,
&nbsp;stations.state,
&nbsp;temp_min.value as temp_min_value,
&nbsp;temp_max.value as temp_max_value,
&nbsp;precipitation.value as precipitation_value

FROM {parameter} base
INNER JOIN {parameter} temp_min ON base.id = temp_min.id AND base.date = temp_min.date
INNER JOIN {parameter} temp_max ON base.id = temp_max.id AND base.date = temp_max.date
INNER JOIN {parameter} precipitation ON base.id = precipitation.id AND base.date = precipitation.date
INNER JOIN bigquery-public-data.ghcn_d.ghcnd_stations stations ON base.id = stations.id

WHERE temp_min.element = 'TMIN' AND temp_max.element = 'TMAX' AND precipitation.element = 'PRCP' AND stations.id = 'USW00013743'

ORDER BY base.id, base.date
"""
</pre><p> </p><p>Also the final line where you call <code>df.append(df_temp)</code> , please make sure you save it in a variable, because right now it is only printing the result. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Interesting. I like this. I will convert to f-string</p>